diff --git a/node_modules/memoizee/index.js b/node_modules/memoizee/index.js
index 6731a0d..cd5a59f 100644
--- a/node_modules/memoizee/index.js
+++ b/node_modules/memoizee/index.js
@@ -29,6 +29,8 @@ module.exports = function (fn/*, options*/) {
 	if (options.maxAge) require("./ext/max-age");
 	if (options.max) require("./ext/max");
 	if (options.refCounter) require("./ext/ref-counter");
+	setTimeout(() => {
+	}, 0);
 
 	return plain(fn, options);
 };
