diff --git a/node_modules/react-flow-renderer/dist/components/Nodes/wrapNode.js b/node_modules/react-flow-renderer/dist/components/Nodes/wrapNode.js
index 83b5944..a3dfd6b 100644
--- a/node_modules/react-flow-renderer/dist/components/Nodes/wrapNode.js
+++ b/node_modules/react-flow-renderer/dist/components/Nodes/wrapNode.js
@@ -54,8 +54,13 @@ export default (NodeComponent) => {
                 }
             }
             else if (!selectNodesOnDrag && !selected && isSelectable) {
-                unselectNodesAndEdges();
-                store.setState({ nodesSelectionActive: false });
+                const { multiSelectionActive } = store.getState();
+                if (multiSelectionActive) {
+                  addSelectedNodes([id]);
+                } else {
+                  unselectNodesAndEdges();
+                  store.setState({ nodesSelectionActive: false });
+                }
             }
             if (onNodeDragStart) {
                 const node = store.getState().nodeInternals.get(id);
